build/test/headless.js

```diff
@@ -0,0 +1,118 @@
+/*global window*/
+const puppeteer = require('puppeteer');
+const getTestPaths = require('./get-test-paths');
+
+(async () => {
+	const testPaths = await getTestPaths();
+
+	if (!testPaths || !testPaths.length) {
+		console.error(`No test paths are specified to run in headless mode.`);
+	}
+
+	/**
+	 * Launch puppeteer
+	 */
+	const browser = await puppeteer.launch({
+		headless: true,
+		args: ['--no-sandbox', '-–disable-setuid-sandbox'],
+		ignoreHTTPSErrors: true,
+		devtools: true
```

Comment:
Why do we want the devtools?

Reply:
devtools can't be opened in headless mode. So, one of these is getting ignored.

Reply:
I used it for debugging locally & left it by mistake.